<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSchedule</class>
 <widget class="QWidget" name="FormSchedule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
     <item>
      <widget class="ScheduleWidget" name="scheduleWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QToolButton" name="toolButtonUndo">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonRedo">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Грануляция</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEditPrecision">
         <property name="minimumTime">
          <time>
           <hour>0</hour>
           <minute>1</minute>
           <second>0</second>
          </time>
         </property>
         <property name="displayFormat">
          <string>HH:mm</string>
         </property>
         <property name="time">
          <time>
           <hour>0</hour>
           <minute>1</minute>
           <second>0</second>
          </time>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labelCurrentObject">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelCurrentTime">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>00:00</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="Ui.qrc">
     <normaloff>:/Icons/Undo.png</normaloff>:/Icons/Undo.png</iconset>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
   <property name="toolTip">
    <string>Отменить действие</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="Ui.qrc">
     <normaloff>:/Icons/Redo.png</normaloff>:/Icons/Redo.png</iconset>
   </property>
   <property name="text">
    <string>Повторить</string>
   </property>
   <property name="toolTip">
    <string>Повторить действие</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScheduleWidget</class>
   <extends>QWidget</extends>
   <header>ScheduleWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Ui.qrc"/>
 </resources>
 <connections/>
</ui>
